snooty/rstspec.toml

```diff
@@ -784,16 +642,16 @@ argument_type = "string"
 inherit = "_landing-base"
 content_type = "block"
 
-[directive."landing:button"]
+[directive."mongodb:button"]
 help = "Make a button."
 example = """.. button:: ${1: string}
 :uri: ${2:/path or url}"""
 argument_type = "string"
-content_type = "_landing-block"
+content_type = "block"
 options.uri = { type = ["path", "uri"], required = true }
 
-[directive."landing:card-group"]
-inherit = "_landing-block"
+[directive."mongodb:card-group"]
```

Comment:
We've now got mongodb:card-group and also [directive.card-group](https://github.com/mongodb/snooty-parser/pull/314/files#diff-8c18bfff17bea8d40e6b0338264e6176ec2b78618336920fa42b71c5409dd88fR458) on line 458 -- do we want/need both?

Reply:
Good question - I left in `directive.card-group` because it is [in-use](https://github.com/mongodb/docs-bi-connector/blob/4460b478bc2f68dc4bc2c52968bf54ce0e36596d/source/client-applications.txt#L32) by `docs-bi-connector`. However, because the `mongodb` domain is searched first, and because we have a defined `mongodb:card-group`, `docs-bi-connector` never finds `std:card-group` when parsed, and just gives a warning about the specified `:type:` option (which is in the `std:card-group` but not the `mongodb:card-group`). So, I can remove this line - but it might be a good idea to figure out a way to more gracefully transition the `bi-connector` cards to the new `card-group` & `card` type